### PR TITLE
chore: pubspec homepage, repository, issue_tracker, and documentation links

### PR DESCRIPTION
**Pull request by renancaraujo**
## Status

**READY**

## Description

Pub doesn't infer homepage as repository anymore. Also, since we have now a docs site, add that to allow people go directly from pub.dev to the docs. 

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [x] 🗑️ Chore

### CODE REVIEW

File: packages/dart_frog/pubspec.yaml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 name: dart_frog
 description: A fast, minimalistic backend framework for Dart built by Very Good Ventures.
 version: 0.0.1-dev.10
-homepage: https://github.com/VeryGoodOpenSource/dart_frog
+homepage: https://dartfrog.vgv.dev/
+repository: https://github.com/VeryGoodOpenSource/dart_frog
+issue_tracker: https://github.com/VeryGoodOpenSource/dart_frog/issues
+documentation: https://dartfrog.vgv.dev/
```
Comment (felangel) [inline packages/dart_frog/pubspec.yaml:7]:
```suggestion
documentation: https://dartfrog.vgv.dev
```

File: packages/dart_frog_cli/pubspec.yaml [hunk 2]
```diff
@@ -1,7 +1,10 @@
 name: dart_frog_cli
 description: The official command line interface for Dart Frog. Built by Very Good Ventures.
 version: 0.0.2-dev.8
-homepage: https://github.com/VeryGoodOpenSource/dart_frog
+homepage: https://dartfrog.vgv.dev/
+repository: https://github.com/VeryGoodOpenSource/dart_frog
+issue_tracker: https://github.com/VeryGoodOpenSource/dart_frog/issues
+documentation: https://dartfrog.vgv.dev/
```
Comment (felangel) [inline packages/dart_frog_cli/pubspec.yaml:7]:
```suggestion
documentation: https://dartfrog.vgv.dev
```

File: packages/dart_frog_gen/pubspec.yaml [hunk 3]
```diff
@@ -1,7 +1,10 @@
 name: dart_frog_gen
 description: Code generation tooling for Dart Frog. Built by Very Good Ventures.
 version: 0.0.2-dev.4
-homepage: https://github.com/VeryGoodOpenSource/dart_frog
+homepage: https://dartfrog.vgv.dev/
+repository: https://github.com/VeryGoodOpenSource/dart_frog
+issue_tracker: https://github.com/VeryGoodOpenSource/dart_frog/issues
+documentation: https://dartfrog.vgv.dev/
```
Comment (felangel) [inline packages/dart_frog_gen/pubspec.yaml:7]:
```suggestion
documentation: https://dartfrog.vgv.dev
```

File: packages/dart_frog/pubspec.yaml [hunk 4]
```diff
@@ -1,7 +1,10 @@
 name: dart_frog
 description: A fast, minimalistic backend framework for Dart built by Very Good Ventures.
 version: 0.0.1-dev.10
-homepage: https://github.com/VeryGoodOpenSource/dart_frog
+homepage: https://dartfrog.vgv.dev/
```
[the remark this answers is not in the document]
Comment (renancaraujo) [reply packages/dart_frog/pubspec.yaml:4]: You mean `repository`?

[Docs says](https://dart.dev/tools/pub/pubspec#homepage) that this should be pointed to a "website for this package". 

Some packages that dos this:
- [Community plus](https://pub.dev/packages/connectivity_plus) packages 
- [flutter_bloc](https://pub.dev/packages/flutter_bloc)

[the remark this answers is not in the document]
Comment (felangel) [reply packages/dart_frog/pubspec.yaml:4]: Yeah I notice some inconsistencies in bloc haha (pkg:bloc vs pkg:flutter_bloc) but I generally agree. Thoughts on making the documentation url point to https://dartfrog.vgv.dev/docs/overview?

Comment (felangel) [inline packages/dart_frog/pubspec.yaml:4]:
```suggestion
homepage: https://dartfrog.vgv.dev/
```
```suggestion
homepage: https://dartfrog.vgv.dev
```

File: packages/dart_frog_gen/pubspec.yaml [hunk 5]
```diff
@@ -1,7 +1,10 @@
 name: dart_frog_gen
 description: Code generation tooling for Dart Frog. Built by Very Good Ventures.
 version: 0.0.2-dev.4
-homepage: https://github.com/VeryGoodOpenSource/dart_frog
+homepage: https://dartfrog.vgv.dev/
```
Comment (felangel) [inline packages/dart_frog_gen/pubspec.yaml:4]:
```suggestion
homepage: https://dartfrog.vgv.dev
```

File: packages/dart_frog_cli/pubspec.yaml [hunk 6]
```diff
@@ -1,7 +1,10 @@
 name: dart_frog_cli
 description: The official command line interface for Dart Frog. Built by Very Good Ventures.
 version: 0.0.2-dev.8
-homepage: https://github.com/VeryGoodOpenSource/dart_frog
+homepage: https://dartfrog.vgv.dev/
```
Comment (felangel) [inline packages/dart_frog_cli/pubspec.yaml:4]:
```suggestion
homepage: https://dartfrog.vgv.dev
```

File: packages/dart_frog/pubspec.yaml (hunk 4 above)
Comment (felangel) [inline packages/dart_frog/pubspec.yaml:4]:
```suggestion
homepage: https://dartfrog.vgv.dev
```

[the remark this answers is not in the document]
Comment (renancaraujo) [reply packages/dart_frog/pubspec.yaml:4]: Not against it, just thought the home would be more future proof

**Review by felangel — APPROVED** [review]
LGTM 🎉

**Pull request merged**